TI-reports/2025/2025-Q1-AI-ML-WG.md

```diff
@@ -0,0 +1,62 @@
+# 2024 Q3 AI/ML WG and model signing project
+
+## Overview
+
+We continue our mission to enable security for AI and use AI for security of open source. The community is healthy, although there are multiple similar groups in various other organisations. This means we have to dedicate time to create interlocks between these groups, to share the knowledge and developments, and make sure work is not duplicated. It also means we have to find the niche where this group can bring added value -- we are currently engaging in determining this. At the same time, we continue work on model signing and experiments towards covering more of the ML supply chain.
+
+## General WG updates
+
+### Purpose
+
+We aim for the WG to be the central place to collect all information regarding ML and security, but we are also planning to deliver technical initiatives, such as the model signing project.
+
+### Current Status
+
+The AI/ML working group now has incubating status.
+
+We have updated the list of interlocks with other working groups in the AI security space. We have started some work on a taxonomy of all these groups and what areas we can collaborate on, what updates we can give/receive, what areas we can work on.  This helps in identifiying gaps and prevent duplication and dillution/fracturing of communities.
+
+There is ongoing work on [mapping AI development lifecycle on the MLSecOps diagram from Ericson](https://github.com/ossf/ai-ml-security/issues/16). Goal is to create an LLMSecOps version as one of the technical outputs of this group in this year.
+
+### Up Next
+
+The group is planning for significant projects to work on in 2025, to [bring
+added value](https://github.com/ossf/ai-ml-security/issues/26).
+
+### Questions/Issues for the TAC
+
+None
+
+## Model signing project
+
+### Purpose
+
+Serve as a central place to collate any recommendation for using AI securely ("security for AI") and using AI to improve security of other products ("AI for security").
+
+The model signing project aims to create a cryptographic signing specification for AI models, taking into account difficulties that arise when trying to protect the integrity of large blobs.
```

Comment:
Where is that spec actually? Could you please provide a link?
I looked for it and couldn't find any link to it from the WG repo or meeting notes.
Thanks.

Reply:
We're still working on it, based on the implementation on https://github.com/sigstore/model-transparency

There are no formal docs at the moment but we should get some befor ethe stable release.

Reply:
Ok, that makes sense, thanks for the clarification but please change the text below to make it clear what the "1.0 release" refers to. I thought it was the spec.
Thanks.

Reply:
Sounds good. Added some clarification text